saleor/graphql/meta/mutations.py

```diff
@@ -114,6 +115,10 @@ def perform_model_extra_actions(cls, root, info, **data):
         """Run extra metadata method based on mutating model."""
         type_name, _ = graphene.Node.from_global_id(data["id"])
         if MODEL_EXTRA_METHODS.get(type_name):
+            if type_name == product_models.Product.__name__:
+                data[
+                    "qs"
+                ] = product_models.Product.objects.prefetched_product_for_webhook()
             instance = cls.get_instance(info, **data)
```

Comment:
Why you can't call this prefetch inside the function responsible for Product extra actions?